src/MagFieldLFT.jl

```diff
@@ -1097,10 +1120,16 @@ term_sym = Dict(0 => "S",
                 4 => "G",
                 5 => "H",
                 6 => "I",
-                7 => "J",
-                8 => "K",
-                9 => "L",
-                10 => "M")
+                7 => "K",
```

Comment:
Do you have a reference for assigning these labels to angular momentum quantum numbers?
(I assumed they would just continue alphabetically, and did not know that e.g. J was left out)

Reply:
The J is not included in the term labeling. You can crosscheck on wikipedia (https://en.wikipedia.org/wiki/Term_symbol) and/or at: https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Spectroscopy/Electronic_Spectroscopy/Spin-orbit_Coupling/Atomic_Term_Symbols